lib/registries/addon/components/registration-list/card/template.hbs

```diff
@@ -24,7 +24,7 @@
             >
                 {{@registration.title}}
             </h3>
-            <ReviewActionsList @reviewActions={{this.reviewActions}} />
+            <ReviewActionsList @reviewActions={{this.reviewActions}} @registration={{@registration}}/>
```

Comment:
I was on the fence about whether to let the component handle fetching the review actions, mostly because the component didn't need to know what the `registration` is. Now that pass `registration` as a parameter to the `ReviewActionsList` component, does it make sense to move `fetchActions` to the component as well?